student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Student < Person
+  def initilize(classroom)
+    super()
```

Comment:
- [ ] **Required:** To get the required input from the parent class, restructure your `initialize()` as below:

```
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```